Ports/bison/package.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=bison
+version=1.25
+useconfigure=true
+configopts="--prefix=$SERENITY_ROOT/Build/Root/usr/local"
+files="http://ftp.gnu.org/gnu/bison/bison-${version}.tar.gz bison-${version}.tar.gz"
+auth_type="md5"
+auth_import_key="65f577d0f8ffaf61ae21c23c0918d225"
```

Comment:
That's not how hash validation works, `auth_import_key` is for `auth_type=sig` :)

Also, let's use HTTPS here.

```suggestion
files="https://ftp.gnu.org/gnu/bison/bison-${version}.tar.gz bison-${version}.tar.gz 65f577d0f8ffaf61ae21c23c0918d225"
auth_type="md5"
```

Reply:
Thanks for the info. I wasn't quite sure on how to handle them.